app/controllers/batches_controller.rb

```diff
@@ -290,25 +290,24 @@ def print_barcodes
     end
   end
 
+  # Handles printing of the worksheet
   def print # rubocop:todo Metrics/AbcSize
```

Comment:
By default, pipelines fell back to a `detailed_worksheet`, which I discovered while looking for other users of `request.value_for`. However this worksheet was throwing exceptions, and would have been for quite some time. Furthermore, its contents made no sense in the context of the only place it was used, the Pacbio pipelines.

So it was clear that it was unused. Set the default to false instead, and added logic to redirect when there's no template. (Also hide the link)

Then rubocop complained, so did a bit of refactoring. (@workflow was unused)